packages/js-dpp/lib/identity/validation/validatePublicKeysFactory.js

```diff
@@ -38,17 +40,24 @@ function validatePublicKeysFactory(validator, bls) {
    * @typedef validatePublicKeys
    *
    * @param {RawIdentityPublicKey[]} rawPublicKeys
+   * @param {boolean} [mustBeEnabled]
    *
    * @return {ValidationResult}
    */
-  function validatePublicKeys(rawPublicKeys) {
+  function validatePublicKeys(rawPublicKeys, mustBeEnabled = false) {
```

Comment:
We usually use options otherwise magic flags are not readable and it might create a sequence of true/false if we need to add more.

Reply:
answer for the first question yes. e.g. ids might be non unique